### PR TITLE
Add Environment Variable for Audience

### PR DESCRIPTION
**Pull request by blijblijblij**
Nice project, was able to put oidc in front of some internal services unable to do oidc.

Just my two cents.

Regards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ This repository builds a Docker Image that protects an upstream server using [Ok
 - `CLIENT_SECRET` - The Client Secret be found on the 'General' tab of the Web application that you created earlier in the Okta Developer Console
 - `ISSUER` - Issuer is the URL of the authorization server that will perform authentication. All Developer Accounts have a 'default' authorization server. The issuer is a combination of your Org URL (found in the upper right of the console home page) and /oauth2/default. For example, `https://xxxxx.oktapreview.com/oauth2/default`
 - `LOGIN_REDIRECT_URL` - The Login Redirect URL can be found on the 'General' tab of the Web application that you created earlier in the Okta Developer Console
+- `CLIENT_AUDIENCE` - The Audience can be Found in the 'Settings' of the Authorization services
```
Comment (caleblloyd) [inline README.md]: This should just be `AUDIENCE`

File: server.go [hunk 2]
```diff
@@ -41,6 +41,11 @@ type jwtResponse struct {
 
 func getConfig() *config {
 	//Populate config from env vars
+	clientAudience := os.Getenv("CLIENT_AUDIENCE")
```
Comment (caleblloyd) [inline server.go]: `AUDIENCE`

File: server.go [hunk 3]
```diff
@@ -97,7 +102,7 @@ func getConfig() *config {
 
 	//Initialize validator
 	toValidate := map[string]string{}
-	toValidate["aud"] = "api://default"
+	toValidate["aud"] = clientAudience
```
Comment (caleblloyd) [inline server.go]: `audience`

Comment (blijblijblij) [reply server.go]: Cool, suggestions made sense, updated them, please review and kind regards, Rogier

**Comment by caleblloyd** [comment]
Thanks for the contribution!

**Comment by blijblijblij** [comment]
Cool, thx!

**Pull request merged**